control-plane/plugin/src/resources/node.rs

```diff
@@ -142,17 +152,28 @@ impl Cordoning for Node {
                 }
                 OutputFormat::None => {
                     // In case the output format is not specified, show a success message.
-                    let cordon_labels = node
-                        .into_body()
-                        .spec
-                        .map(|node_spec| node_spec.cordon_labels)
-                        .unwrap_or_default();
-                    if cordon_labels.is_empty() {
+                    let mut cordon_labels: Vec<String> = vec![];
+                    let mut drain_labels: Vec<String> = vec![];
+                    match node.clone().into_body().spec.unwrap().cordondrainstate {
```

Comment:
need to handle case when spec is None?

Reply:
fixed, and took the opportunity to simplify the code